ontology/yaml/resources/HVAC/entity_types/FAN.yaml

```diff
@@ -605,6 +605,15 @@ FAN_SS_NONCANONICAL_5:
   - dryer_run_status_5
   - filter_differential_pressure_sensor
 
+FAN_SS_NONCANONICAL_6:
+  description: "Non-canonical fan which uses differential airflow measured in a space to provide pressure balancing."
+  is_canonical: false
+  uses:
+  - run_command
```

Comment:
please remove run_command and run_status as individual fields and instead put implements: -SS

Reply:
+1